src/visualization/interpolate.jl

```diff
@@ -176,6 +176,78 @@ function unstructured_2d_to_3d(unstructured_data, coordinates, levels,
   return unstructured_data, new_coordinates, new_levels, center_level_0
 end
 
+# Convert 2d unstructured data to 1d stripe.
+function unstructured_1d_to_2d(original_nodes, unstructured_data, stripe_axis, stripe_axis_intercept)
```

Comment:
This should be named differently, or am I missing something?
```suggestion
function unstructured_2d_to_1d(original_nodes, unstructured_data, stripe_axis, stripe_axis_intercept)
```

Reply:
Yes, I changed this to: 2d_to_1d and 3d_to_2d for the 2D case aswell.